gen/definitions/evpn_ethernet_segment.yaml

```diff
@@ -0,0 +1,40 @@
+---
+name: EVPN Ethernet Segment
+path: Cisco-IOS-XE-native:native/l2vpn/Cisco-IOS-XE-l2vpn:evpn_cont/evpn-ethernet-segment/evpn/ethernet-segment=%v
+test_tags: [C9000V]
+doc_category: EVPN
+attributes:
+  # l2vpn evpn ethernet-segment 1
+  - yang_name: es-value
+    example: 1
+  # df-election wait-time 3
+  - yang_name: df-election/wait-time
+    example: 3
+  # redundancy all-active
+  - yang_name: redundancy/redundancy-choice/all-active/all-active
+    xpath: redundancy/all-active
+    tf_name: redundancy_all_active
```

Comment:
The `tf_name` should be derived from the `xpath` (if defined), hence it should not be required here.